large timeout values trigger compilation warning.

When setting 
```plain
#define SCREENSAVER_B_MAX_TIME_SEC 3600
```

I get this warning

```plain
/mnt/src/defaults.c:44:59: warning: integer overflow in expression of type 'int' results in '-694967296' [-Woverflow]
   44 |                 .max_time_us = SCREENSAVER_B_MAX_TIME_SEC * 1000000,
```

It's an easy fix, enforce 64 bit precision at assignment:
```c
                .max_time_us = (uint64_t)SCREENSAVER_B_MAX_TIME_SEC * 1000000,
```

This needs to be done for 
- `SCREENSAVER_(A|B)_IDLE_TIME_SEC`
- `SCREENSAVER_(A|B)_MAX_TIME_SEC`

--- a/src/defaults.c
+++ b/src/defaults.c
@@ -20,8 +20,8 @@ const config_t default_config = {
             .screensaver = {
                 .mode = SCREENSAVER_A_MODE,
                 .only_if_inactive = SCREENSAVER_A_ONLY_IF_INACTIVE,
-                .idle_time_us = SCREENSAVER_A_IDLE_TIME_SEC * 1000000,
-                .max_time_us = SCREENSAVER_A_MAX_TIME_SEC * 1000000,
+                .idle_time_us = (uint64_t)SCREENSAVER_A_IDLE_TIME_SEC * 1000000,
+                .max_time_us = (uint64_t)SCREENSAVER_A_MAX_TIME_SEC * 1000000,
             }
         },
     .output[OUTPUT_B] =
@@ -40,8 +40,8 @@ const config_t default_config = {
             .screensaver = {
                 .mode = SCREENSAVER_B_MODE,
                 .only_if_inactive = SCREENSAVER_B_ONLY_IF_INACTIVE,
-                .idle_time_us = SCREENSAVER_B_IDLE_TIME_SEC * 1000000,
-                .max_time_us = SCREENSAVER_B_MAX_TIME_SEC * 1000000,
+                .idle_time_us = (uint64_t)SCREENSAVER_B_IDLE_TIME_SEC * 1000000,
+                .max_time_us = (uint64_t)SCREENSAVER_B_MAX_TIME_SEC * 1000000,
             }
         },
     .enforce_ports = ENFORCE_PORTS,

